plataform_front_end/src/components/loader/loader.css

```diff
@@ -0,0 +1,37 @@
+@import url('https://fonts.googleapis.com/css?family= Baloo Da');
+
+body{
+    position: relative;
+    height: 100vh;
+ 
+    background: #3A0088;
+    background-image: url("../img/loaderImg.png");
+    background-size: cover;
+    background-repeat: no-repeat; 
+}
+
+p{
+
+   
+    color: #FFBD39;
+    -webkit-text-stroke-width: 3px; 
+    -webkit-text-stroke-color: #3a0088; 
+    
+    font-family: Baloo Da;
+    font-style: normal;
+    font-weight: bold;
+    font-size: calc(4em + 7vw); 
+    
+    display: flex;
+    /* flex-direction: column; */
+    justify-content: center;
+
+}
+
+.homer {
```

Comment:
You may have some problems with these selectors/classes that are not that specifics. We can talk about it so I can explain it better.